core/src/main/java/greencity/controller/EventController.java

```diff
@@ -0,0 +1,124 @@
+package greencity.controller;
+
+import greencity.annotations.CurrentUser;
+import greencity.constant.AppConstant;
+import greencity.dto.event.EventDto;
+import greencity.annotations.ImageListSizeValidation;
+import greencity.annotations.ImageSizeValidation;
+import greencity.annotations.ImageValidation;
+import greencity.constant.HttpStatuses;
+import greencity.dto.event.EventEditDto;
+import greencity.dto.user.UserVO;
+import greencity.exception.handler.MessageResponse;
+import greencity.service.EventService;
+import io.swagger.v3.oas.annotations.Operation;
+import greencity.constant.SwaggerExampleModel;
+import greencity.dto.event.EventCreationDtoRequest;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import java.util.Set;
+import java.util.List;
+
+
+@Validated
+@RestController
+@RequestMapping("/events")
+@RequiredArgsConstructor
+public class EventController {
+    private final EventService eventService;
+
+
+    @Operation(summary = "Create new event.")
+    @ResponseStatus(value = HttpStatus.CREATED)
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "201", description = "Event created successfully"),
+            @ApiResponse(responseCode = "401", description = "Unauthorized"),
+            @ApiResponse(responseCode = "403", description = "Bad Request")
```

Comment:
If it should have been 403, it should have been forbidden the description
And if there should be a bad request, then it is 404

Reply:
Thank you for helping me and making me think. 404 Not Found is used when a resource or page does not exist. For a bad request, we should return 400, and for a missing resource, we should return 404.